styles.css

```diff
@@ -401,3 +415,134 @@ body {
 .end-Line {
   padding-top: 10px;
 }
+
+.navigation ul {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .Hamburger,
```

Comment:
- [ ] You have respected the requirement on the media query size but your design is not what we expected, see this ⬇️, it is not responsive, please update the CSS media query style to make your app as responsive, your card, your texts, spaces and all the rest. take care of comparing the desing on the right which is your with the original design given on the left .

![5](https://user-images.githubusercontent.com/77843050/213786966-18ee0c32-a5a6-4ee1-aabd-6180a81fa229.gif)